todo_app/test/test_viewmodel.py

```diff
@@ -16,3 +16,6 @@ def test_done_items():
     viewmodel = ViewModel([], [], [item])
     assert viewmodel.done_lists == [item]
 
+# 1. A user can add a new item to the to do list - e2e
+# 2. The app can handle a response from Trello's API - integration
+# 3. The item sets a completed_at field to the current time when it is marked as 'Done' - unit
```

Comment:
This looks like it's probably comments for you rather than future users, so worth clearing out?